development/packaging.md

```diff
@@ -73,6 +73,19 @@ they cannot access the organisation-wide secret we have in place.
 2. Add it as a Repository Secret to the Private Repo (https://github.com/flowforge/<repo-name>/settings/secrets/actions)
    with the name `PROJECT_ACCESS_TOKEN`
 
+### Labels
+
+We have a standard set of labels that should be applied to all repositories. This
+ensures we have a consistent approach to planning and tracking of work.
+
+ - Type: `epic`, `story`, `task`, `bug`
+ - Sizing: `1`, `2`, `3`, `5`, `8`, `13`
+ - Area: `area:docs`, `area:db`, `area:migration`, `area:frontend`, `area:api`, `area:device`
+ - Priority: `priority:high`, `priority:medium`, `priority:low`
```

Comment:
Is medium redundant and unlikely to be used? i.e. the default is medium, high means do this first, low means do this last.

Reply:
That is certainly possible. I'm also away that priority will change over time - so I'm wary of having too many labels that have to be constant juggled.

Whilst I've included medium/low in the list, my gut feeling is we'll mainly use `high` as a 'must-have' flag. But I'd rather have medium/low available to use if we find we need them... and then get rid of them later if not.